doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,230 @@
+### 5.2.1 General
+The Descriptor File defines the unique configuration required by installer in a common schema. 
+It would specialize the installer type per user's implementation requirements.
+It would be validated at the very beginning of the deployment.
+It's installer's responsibility to translate the descriptor file to adapt with its own configuration. 
+Thanks to the descriptor file, the NFVi infrastructure deployment could be completed in one step run.
 
 
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.gen.ins.01` | Installer | Installer **should** accept a descriptor file to finish deployment.|
+| `req.gen.ins.02` | Installer | Installer implementation **must** validate the descriptor file with schema.|
+| `req.gen.ins.03` | Installer | Any existing installer implementation **may** need adaption for the descriptor file. |
+| `req.gen.ins.04` | Installer | Installer **may** support reporting the deplolyment progress status.|
+| `req.gen.des.01` | Descriptor | Descriptor file **should** include hardware resource configuration, software configuraiton.|
+| `req.gen.des.02` | Descriptor | Descriptor file **may** include additional extending configuraiton.|
+
+<a name="5.2.2"></a>
+### 5.2.2 Additional
+Depends xxx.
+
 <a name="5.3"></a>
-## 5.3 Heading
+## 5.3 Descriptor definition
+There should be a Descriptor File definition, which used by installer as input of necessary configuration.
+Mandatory and optional definition should be defined.
+ 
+<a name="5.3.1"></a>
+### 5.3.1 Hardware resource information
+The support of different workload types, each with different compute, storage, network requirements which needs kinds of hardware configuration template, we name this as server template.
+Besides it may include optional information such area name, data center name etc.
+
+| Field # | type | mandatory? | Instruction |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| area_name | String | Yes | data center area name |
+| area_center_name | String | Yes | data center name, may compliance with a naming rule.  |
+| room_name | String | Yes | rome static data, helpful in locating issue occurred. |
+| city | String | Yes |  |
+| resourece_pool_name | String | Yes |  |
+| outband_network_segment | String | Yes |  |
+| server_templates | List | Yes |  |
+| server_infos | List | Yes |  |
+
+<p align="center"><b>Table 5-3-1:</b> Hardware resource description.</p>
+
+### 5.3.2 Server template
+The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
```

Comment:
**Query:** Does NIC binding needs to be referred here ? AFAIK, NIC bonding is configured at the OS/kernel level. Since this section talks about hardware, NIC bonding may not be a part of this section.
(Correct me if I am wrong)

Reply:
Actually, I suppose not all server template include NIC binding case, so may be better to move under server than server template.

Reply:
Sounds good.

Reply:
Yes, it should be optional. I agree.